css/bookliststyle.css

```diff
@@ -0,0 +1,205 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,700;0,900;1,900&display=swap");
+@import url("https://weloveiconfonts.com/api/?family=entypo");
+
+body {
+  margin: 0;
+  padding: 0;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  color:white;
+  background-image: url("images/book_background.jpg");

```

Comment:
The url should be `../images/book_background.jpg`

Reply:
Thank you. Updated it